src/satellite/client.ts

```diff
@@ -142,8 +142,12 @@ export class SatelliteClient extends EventEmitter implements Client {
 
   enqueueTransaction(transaction: Transaction): void | SatelliteError {
     if (this.outbound.isReplicating != ReplicationStatus.ACTIVE) {
-      throw new SatelliteError(SatelliteErrorCode.REPLICATION_NOT_STARTED,
-        "enqueuing a transaction while outbound replication has not started")
+
+      // XXX
```

Comment:
What's actually going on here?

Reply:
I noticed this but think that we can look into it when we're actually implementing the offline mode.

Leaving a comment: "Client should not allow queueing transactions when Satellite is working in offline mode"